bin/pc2installsandbox

```diff
@@ -0,0 +1,115 @@
+JUDGE_GROUP="$2"
+if ! getent passwd "$JUDGE_USER" >/dev/null 2>&1
+then
+   echo $0: No such user $JUDGE_USER.  The pc2user must be a valid login id.
+   exit $FAIL_BAD_USER_GROUP
+fi
+if ! getent group "$JUDGE_GROUP" >/dev/null 2>&1
+then
+   echo $0: No such group $JUDGE_USER.  The pc2group must be a valid group in /etc/group.
+   exit $FAIL_BAD_USER_GROUP
+fi
+
+# make sure CGroups v2 is installed
+echo Verifying CGroup v2 installation...
+if [ ! -f "$CGROUP_PATH/cgroup.controllers" ]; then
+   echo $0: "It appears CGroups v2 is not installed (missing file 'cgroup.controllers' in $CGROUP_PATH)."
+   echo CGroups v2 must be installed before a PC2 sandbox can be installed
+   echo "(or if CGroups v2 is installed elsewhere from $CGROUP_PATH, edit the CGroup Path in $0)."
+   exit $FAIL_MISSING_CGROUP_CONTROLLERS_FILE
+fi
+
+if [ ! -f "$CGROUP_PATH/cgroup.subtree_control" ]; then
+   echo $0: "It appears CGroups v2 is not installed (missing file 'cgroup.subtree_control' in $CGROUP_PATH)."
+   echo CGroups v2 must be installed before a PC2 sandbox can be installed
+   echo "(or if CGroups v2 is installed elsewhere from $CGROUP_PATH, edit the CGroup Path in $0)."
+   exit $FAIL_MISSING_CGROUP_SUBTREE_CONTROL_FILE
+fi
+
+# make sure the cpu and memory controllers are enabled
+
+if ! grep -q -F "cpu" "$CGROUP_PATH/cgroup.subtree_control"; then
+   echo $0: cgroup.subtree_control in $CGROUP_PATH does not enable cpu controller
+#   echo $0: Perhaps you have: echo +cpu ">$CGROUP_PATH/cgroup.subtree_control"
+   echo $0: Executing: echo +cpu ">$CGROUP_PATH/cgroup.subtree_control"
+   echo +cpu > $CGROUP_PATH/cgroup.subtree_control
+fi
+
+if ! grep -q -F "memory" "$CGROUP_PATH/cgroup.subtree_control"; then
+   echo $0: cgroup.subtree_control in $CGROUP_PATH does not enable memory controller
+   exit $FAIL_MEMORY_CONTROLLER_NOT_ENABLED
+fi
+
+
+# we seem to have a valid CGroup installation
+echo ...done.
+
+# First, remove any existing remnants
+for dir in `ls $PC2_CGROUP_PATH/sandbox* 2>/dev/null`
+do
+	if test -d "$dir"
+	then
+		DEBUG echo Removing existing $dir
+		rmdir "$dir"
+	fi
+done
+if test -d $PC2_CGROUP_PATH
+then
+	rmdir $PC2_CGROUP_PATH
+fi
+
+# Errors will cause script to stop
+set -e
+
+# Create a pc2 cgroup to be managed by the JUDGE user and group 
+echo Creating PC2 CGroup structure...
+sudo mkdir $PC2_CGROUP_PATH
+sudo chown $JUDGE_USER:$JUDGE_GROUP $PC2_CGROUP_PATH
+sudo chmod 775 $PC2_CGROUP_PATH
+
+# allow the JUDGE user and group to manage the pc2 cgroup files and subgroups.
+# note we do NOT change ownership of the "resource control" interface files, like cpu.max;
+# those must remain under control of the PARENT -- root -- cgroup
+# judge accounts use group JUDGE_GROUP so everything must be group write as well.
+sudo chown $JUDGE_USER:$JUDGE_GROUP $PC2_CGROUP_PATH/cgroup.procs
+sudo chown $JUDGE_USER:$JUDGE_GROUP $PC2_CGROUP_PATH/cgroup.subtree_control
+#sudo chown $JUDGE_USER:$JUDGE_GROUP $PC2_CGROUP_PATH/memory.max
```

Comment:
My reaction to the above statement is "why is this commented out?"   If it's unnecessary cruft, it should be removed.  Perhaps a better approach, if the point is to remember something is special about `memory.max`, would be to add a script comment above this line explaining why this line is commented-out...

Reply:
Yeah.  It should be removed. I think I was testing something at the time.